webtau-http-groovy/src/test/groovy/com/twosigma/webtau/http/HttpGroovyTest.groovy

```diff
@@ -296,8 +315,9 @@ class HttpGroovyTest extends HttpTestBase {
             body.should == headerPayload
         }
 
-        http.get("/echo-header", requestHeader, expectations)
-        http.get("/echo-header", [qp1: 'v1'], requestHeader, expectations)
+//        http.get("/echo-header", requestHeader, expectations)
```

Comment:
If these two aren't needed, can we just delete them rather than commenting out please?

Reply:
oops, my bad. I had trouble figuring out test failures and forgot to uncomment these lines. Back now.